modules/doc/content/training/darcy_thermo_mech/step01.md

```diff
@@ -0,0 +1,124 @@
+# Step 1: Creating and Executing a MOOSE Object
```

Comment:
Can you rename this file to step01_moose_object.md. I just like the filename giving some information about what it contains.